packages/common/src/utils/generateReports/generateRootSummaryDataCSV.ts

```diff
@@ -107,27 +100,62 @@ const generateRootSummaryDataCSV = (
     totalCookies += 1;
   });
 
-  const summary = {
-    [I18n.getMessage('totalCookies')]: totalCookies,
-    [I18n.getMessage('totalFirstPartyCookies')]: totalFirstPartyCookies,
-    [I18n.getMessage('totalThirdPartyCookies')]: totalThirdPartyCookies,
-    [I18n.getMessage('analyticsCookies')]: analyticsCookies,
-    [I18n.getMessage('functionalCookies')]: functionalCookies,
-    [I18n.getMessage('marketingCookies')]: marketingCookies,
-    [I18n.getMessage('uncategorizedCookies')]: uncategorisedCookies,
-    [I18n.getMessage('cookiesWithIssues')]: cookiesWithIssues,
-    [I18n.getMessage('analyticsCookiesWithIssues')]: analyticsCookiesWithIssues,
-    [I18n.getMessage('functionalCookiesWithIssues')]:
-      functionalCookiesWithIssues,
-    [I18n.getMessage('marketingCookiesWithIssues')]: marketingCookiesWithIssues,
-    [I18n.getMessage('uncategorizedCookiesWithIssues')]:
-      uncategorisedCookiesWithIssues,
-  };
+  const summary = [
+    totalCookies,
+    totalFirstPartyCookies,
+    totalThirdPartyCookies,
+    analyticsCookies,
+    functionalCookies,
+    marketingCookies,
+    uncategorisedCookies,
+    cookiesWithIssues,
+    analyticsCookiesWithIssues,
+    functionalCookiesWithIssues,
+    marketingCookiesWithIssues,
+    uncategorisedCookiesWithIssues,
+  ];
+
+  return summary.join(',').concat('\r\n');
+};
+export const generateRootSummaryDataCSV = (
+  siteMapAnalysisData: CompleteJson[]
+) => {
+  const extractedData: { [key: string]: CookieTableData } = reshapeCookies(
+    extractReportData(siteMapAnalysisData).landingPageCookies
+  );
+  const headers = [
+    'URL Heading',
```

Comment:
Why `URL Heading` instead of just `URL` ?

Reply:
Fixed in latest commit. [dd317e1](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/849/commits/dd317e1c77f2549884b08f295799731b642062df)